packages/@okta/vuepress-site/docs/guides/oin-lifecycle-mgmt-overview/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Overview of lifecycle management in the OIN
+meta:
+  - name: description
+    content: Learn how to build a lifecycle management app integration for the Okta Integration Network.
+---
+
+Lifecycle management refers to the process of provisioning and deprovisioning application access as a user moves through an organization. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in workforce applications and enforcing corporate security policies in a timely manner.
+
+<div class="half">
+
+![Lifecycle management](/img/oin/scim_lifecycle.png "User lifecycle diagram: 1 - new employee 2 - provision applications 3 - enforce security 4 - update user information 5 - off-board")
+
+</div>
+
+Okta provides two methods to integrate lifecycle management to your application. You can use the [System for Cross-domain Identity Management](/docs/reference/scim/) (SCIM) protocol to manage user accounts from Okta, or you can use the [Workflows Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) to create automated identity processes in [Okta Workflows](https://help.okta.com/okta_help.htm?type=wf).
+
+After completing your Okta lifecycle management integration, publish it in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/) for your customers to benefit secure provisioning automation and meet compliance requirements. The OIN is a collection of pre-built app integrations covering numerous use cases. Join the community that customers trust to exchange secure authentication between users, devices, and applications.
+
+## Why build a lifecycle management integration with Okta?
+
+| &nbsp; | &nbsp; |
+| ------ | ------ |
+| **Automate onboarding/offboarding and win larger customers** | Once an organization reaches a certain size, manually provisioning accounts and getting users set up for success can be time consuming and a huge bottleneck for your product’s adoption. Okta lifecycle management integration automates mundane tasks for your customers which reduces the total cost of ownership and removes barriers to end-user activation. |
+| **Secure access to your application** | Secure access to your application: integrating with Okta ensures that end-users can access your application as soon as they join an organization. Access is automatically removed as they leave the organization to prevent embarrassing data loss and dangling access. |
```

Comment:
```suggestion
| **Secure access to your application** | Secure access to your application. Integrating with Okta ensures that end users can access your application as soon as they join an organization. Access is automatically removed as they leave the organization to prevent embarrassing data loss and dangling access. |
```